### ISSUE TITLE
SSHFS-to-SSHFS

### ISSUE THREAD

**Issue by er1z** [billziss-gh/sshfs-win#60]
Should copying from SSHFS mount to another mount work ok? From Total Commander I get some errors regarding „cannot create a directory”.

**Comment by billziss-gh** [comment]
Yes, it should. Are permissions set up properly on both SSHFS instances?

**Comment by er1z** [comment]
Permissions were ok because transferring data onto particular drives worked correctly.
